js/rightbtc.js

```diff
@@ -429,8 +429,8 @@ module.exports = class rightbtc extends Exchange {
             //   0.036*1e8 === 3599999.9999999995
             // which would get truncated to 3599999 after parseInt
             // which would then be rejected by rightBtc because it's too precise
-            'quantity': parseInt (this.decimalToPrecision (amount * 1e8, ROUND, 0, DECIMAL_PLACES)),
-            'limit': parseInt (this.decimalToPrecision (price * 1e8, ROUND, 0, DECIMAL_PLACES)),
+            'quantity': parseInt (this.decimalToPrecision (amount * 1e8, TRUNCATE, 0, this.precisionMode)),
```

Comment:
This doesn't work, it should be `ROUND`.
To see this yourself, try it with `0.036`

Reply:
@Zinggi yep, thx, to me it looks like a bug in `decimalToPrecision`, let me fix it first...

Reply:
```
> exchange.decimalToPrecision (0.036 * 1000000, ccxt.TRUNCATE, 0, exchange.precisionMode)
'36000'
> exchange.decimalToPrecision (0.036 * 10000000, ccxt.TRUNCATE, 0, exchange.precisionMode)
'360000'
> exchange.decimalToPrecision (0.036 * 100000000, ccxt.TRUNCATE, 0, exchange.precisionMode)
'3599999'
> exchange.decimalToPrecision (0.036 * 1000000000, ccxt.TRUNCATE, 0, exchange.precisionMode)
'36000000'
```

Reply:
wow...

Reply:
@kroitor `0.036 * 100000000` is `3599999.9999999995`, so you're effectively calling:

```
exchange.decimalToPrecision (3599999.9999999995, ccxt.TRUNCATE, 0, exchange.precisionMode)
```

Truncation of `3599999.9999999995` to 0 decimal places is `3599999` (and this is what you're getting as an output). So that doesn't look like a bug to me.